src/components/Modal/index.js

```diff
@@ -0,0 +1,171 @@
+import React, { Component } from 'react';
+import axios from "axios";
+import ReCAPTCHA from "react-google-recaptcha";
+import PropTypes from "prop-types";
+import Modal from "react-modal";
+import Button from "../Button";
+import {EmailInput, TextInput, TextAreaInput} from "../Inputs";
+import MaterialIcon from "../MaterialIcon";
+import {ModalSavedItemList} from "../SavedItem";
+import "./styles.scss"
+
+const MyListModal = (props) => (
+  // TODO: replace captcha key
+  <Modal
+    appElement={props.appElement ? props.appElement : Modal.setAppElement("#root")}
+    isOpen={props.isOpen}
+    onRequestClose={props.toggleModal}
+    className="modal-content"
+    overlayClassName="modal-overlay">
+    <div className="modal-header">
+      <h2 className="modal-header__title">{props.title}</h2>
+      <button className="modal-header__button" aria-label="Close" onClick={props.toggleModal}>
+        <MaterialIcon icon="close"/>
+      </button>
+    </div>
+    <div className="modal-body">
+      <div className="modal-list">
+        <ModalSavedItemList items={props.list} />
+      </div>
+      <div className="modal-form">
+        <form>
+          <div className="modal-form__input-group">
+            {props.inputs}
+          </div>
+          <div className="modal-form__captcha">
+            <ReCAPTCHA
+              sitekey="6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI"
+              onChange={props.handleCaptchaChange} />
+          </div>
+          <div className="modal-form__buttons">
+            {props.buttons}
+            {props.submitError && <p className="modal-error">{props.submitError}</p>}
+          </div>
+        </form>
+      </div>
+    </div>
+  </Modal>
+)
+
+MyListModal.propTypes = {
+  appElement: PropTypes.object,
+  isOpen: PropTypes.bool.isRequired,
+  toggleModal: PropTypes.func.isRequired,
+  handleCaptchaChange: PropTypes.func.isRequired,
+  title: PropTypes.string.isRequired,
+  list: PropTypes.array.isRequired,
+  inputs: PropTypes.node.isRequired,
+  buttons: PropTypes.node.isRequired
+}
+
+export class EmailModal extends Component {
+  // TODO: should buttons be inputs instead?
```

Comment:
I don't think so. [Aren't they functionally the same](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event)? My research tells me it can be harder to style an input to look like a button across all browsers.